fast_llm/engine/checkpoint/external.py

```diff
@@ -35,6 +35,9 @@ def import_params(self, export_values: tuple[typing.Any, ...]) -> tuple[typing.A
 
 @dataclasses.dataclass(kw_only=True)
 class RenameParamConverter(ParamConverter):
+    ignore: bool = False
```

Comment:
Why not just `ignore_missing: bool`? Any scenario where we need to ignore something else, or set a non-default value?